docs/enriching-your-data/available-enrichments/custom-javascript-enrichment/writing/index.md

```diff
@@ -235,6 +235,12 @@ function process(event) {
   ...
 ```
 
+:::note
+
+You might be tempted to update derived entities in a similar way by using `event.setDerived_contexts()`. However, this is not supported — instead, refer to the [Adding extra entities](#adding-extra-entities-to-the-event) section.
```

Comment:
Just for my knowledge, `not supported` here as in "doesn't work", or "please don't"? (Someone somewhere will inevitably try anyway...)